### DISCUSSION TITLE
Student verification issue for university with multiple campus

### DISCUSSION THREAD

**Discussion by FPTSmartEdu** [community/community#63269] [GitHub Education]
### Hi everyone,

Hi, I'm in Vietnam which is very common for university to have multiple campus across the country using the same name. They use "phân hiệu + location" to differentiate and I can't be verified because I'm not near the main campus in Hanoi. Please help!

p/s: I have summited a support ticket for 3 weeks without reply.

**Comment by cesu171** [comment]
Could you tell me if you followed the instructions with [Apply to GitHub Global Campus as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)?

You can just put the school main name there without campus name

**Reply by FPTSmartEdu** [reply]
Because the main campus is in another city so I think they only register the main campus (we have like 5 or 6 campus in different cities). I followed the instruction and it returns "You appear not to be near any campus location for the school you have selected."
